### PR TITLE
chore: Remove double spaces

### PR DESCRIPTION
**Pull request by nschonni**
Looked with a quick regex `\w ( )+\w` and fixed non-code fences

### CODE REVIEW

File: locale/ar/about/privacy.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ layout: about.hbs
...
 
-إقرأ سياسة الخصوصية بتمعن، باستعمال اي  موقع سيتم اعتبارك موافقاً على شروط السياسة. إذا لم توافق على شروط السياسة، سيتم توجيهك لانهاء وصول و استعمال المواقع أو مواد تم الحصول عليها من هذه المواقع.
+إقرأ سياسة الخصوصية بتمعن، باستعمال اي موقع سيتم اعتبارك موافقاً على شروط السياسة. إذا لم توافق على شروط السياسة، سيتم توجيهك لانهاء وصول و استعمال المواقع أو مواد تم الحصول عليها من هذه المواقع.
 
 ## التغييرات على سياسة الخصوصية
 تحتفظ المؤسسة بحق تعديل  و نحديث سياسة الخصوصية هذه من وقت لآخر. في كل مرة يستخدم فيها السمتخدم موقعاً، يتم تطبيق السياسة الحالية للخصوصية. و موازاة مع ذلك، على المستخدم تفقد تاريخ سياسة الخصوصية هذه (التي تظهر في الأعلى) و مراجعة أي تغييرات طرأت منذ النسخة الأخيرة. إذا لم يوافق المستخدم على سياسة الخصوصية، قيجب عليه ألا يستعمل أياً من المواقع و الاستعمال المستمر لاي من هذه المواقع باتباع اي من المراجعات المتعلقة بسياسة الخصوصية يعني القبول بأي تغييرات فيها.
```
Comment (Aissaoui-Ahmed) [inline locale/ar/about/privacy.md]:
```suggestion
تحتفظ المؤسسة بحق تعديل و نحديث سياسة الخصوصية هذه من وقت لآخر. في كل مرة يستخدم فيها السمتخدم موقعاً، يتم تطبيق السياسة الحالية للخصوصية. و موازاة مع ذلك، على المستخدم تفقد تاريخ سياسة الخصوصية هذه (التي تظهر في الأعلى) و مراجعة أي تغييرات طرأت منذ النسخة الأخيرة. إذا لم يوافق المستخدم على سياسة الخصوصية، قيجب عليه ألا يستعمل أياً من المواقع و الاستعمال المستمر لاي من هذه المواقع باتباع اي من المراجعات المتعلقة بسياسة الخصوصية يعني القبول بأي تغييرات فيها.
```

**Comment by XhmikosR** [comment]
LGTM for some reason I can't commit the above suggestion, I get `You do not have permission to push to this repository.`

File: locale/fa/about/index.md [hunk 2]
```diff
@@ -36,7 +36,7 @@ server.listen(port, hostname, () => {
 
 ---
 
-Node.js در طراحی مشابه و تاثیر گرفته است از سیستم‌هایی ماننده  Ruby's
+Node.js در طراحی مشابه و تاثیر گرفته است از سیستم‌هایی ماننده Ruby's
 [Event Machine][] یا Python's [Twisted][].
 Node.js مدل رویداد را کمی به جلوتر می‌برد و  [event loop][] را به عنوان یک ساختار زمان‌بندی به جای یک کتابخانه ارائه می‌کند.
```
Comment (Aissaoui-Ahmed) [inline locale/fa/about/index.md]:
```suggestion
Node.js مدل رویداد را کمی به جلوتر می‌برد و [event loop][] را به عنوان یک ساختار زمان‌بندی به جای یک کتابخانه ارائه می‌کند.
```

File: locale/fa/download/releases.md [hunk 3]
```diff
@@ -5,7 +5,7 @@ modules: "<code>NODE_MODULE_VERSION</code> refers to the ABI (application binary
 ---
 
 ### io.js و Node.js
-به انتشارهای  1.x تا 3.x در گذشته  "io.js" می‌گفتند.
+به انتشارهای 1.x تا 3.x در گذشته  "io.js" می‌گفتند.
```
Comment (Aissaoui-Ahmed) [inline locale/fa/download/releases.md]:
```suggestion
به انتشارهای 1.x تا 3.x در گذشته "io.js" می‌گفتند.
```

File: locale/fa/download/releases.md [hunk 4]
```diff
@@ -5,7 +5,7 @@ modules: "<code>NODE_MODULE_VERSION</code> refers to the ABI (application binary
...
  از آن جایی که آن‌ها بخشی از io.js fork بودند
   از Node.js 4.0.0 خط های انتشارهای قبلی  io.js در   Node.js 0.12.x  کاور شدند که به شکل انتشارهای یک پارجه در آمدند.
```
Comment (Aissaoui-Ahmed) [inline locale/fa/download/releases.md]:
```suggestion
 از Node.js 4.0.0 خط های انتشارهای قبلی io.js در Node.js 0.12.x کاور شدند که به شکل انتشارهای یک پارجه در آمدند.
```

File: locale/fa/index.md [hunk 5]
```diff
@@ -20,4 +20,4 @@ labels:
   newsletter-postfix: "، خبرنامهٔ رسمی NodeJs.org"
 ---
 
-Node.js®   چارچوبی است برای اجرای جاوااسکریپت  ساخته شده بر روی  [موتور جاوااسکریپت کروم](https://v8.dev/).
+Node.js®   چارچوبی است برای اجرای جاوااسکریپت ساخته شده بر روی  [موتور جاوااسکریپت کروم](https://v8.dev/).
```
Comment (Aissaoui-Ahmed) [inline locale/fa/index.md]:
```suggestion
Node.js® چارچوبی است برای اجرای جاوااسکریپت ساخته شده بر روی [موتور جاوااسکریپت کروم](https://v8.dev/).
```

File: locale/fa/security.md [hunk 6]
```diff
@@ -15,7 +15,7 @@ title: امنیت
 
 ### برنامه جایزه پیدا کردن باگ در Node.js
 
-پروژه Node.js در یک برنامه رسمی جایزه برای باگ شرکت می‌کند که برای فعالان امنیتی و  افشاهای عمومی است.
+پروژه Node.js در یک برنامه رسمی جایزه برای باگ شرکت می‌کند که برای فعالان امنیتی و افشاهای عمومی است.
 
 این برنامه از طریق پلتفرم HackerOne مدیریت می‌شود به آدرس  [https://hackerone.com/nodejs](https://hackerone.com/nodejs) برای اطلاعات بیشتر.
```
Comment (Aissaoui-Ahmed) [inline locale/fa/security.md]:
```suggestion
این برنامه از طریق پلتفرم HackerOne مدیریت می‌شود به آدرس [https://hackerone.com/nodejs](https://hackerone.com/nodejs) برای اطلاعات بیشتر.
```

**Review by phillipj — APPROVED** [review]

File: locale/fa/about/index.md [hunk 7]
```diff
@@ -36,9 +36,9 @@ server.listen(port, hostname, () => {
 
 ---
 
-Node.js در طراحی مشابه و تاثیر گرفته است از سیستم‌هایی ماننده  Ruby's
+Node.js در طراحی مشابه و تاثیر گرفته است از سیستم‌هایی ماننده Ruby's
 [Event Machine][] یا Python's [Twisted][].
-Node.js مدل رویداد را کمی به جلوتر می‌برد و  [event loop][] را به عنوان یک ساختار زمان‌بندی به جای یک کتابخانه ارائه می‌کند.
+Node.js مدل رویداد را کمی به جلوتر می‌برد و [event loop][] را به عنوان یک ساختار زمان‌بندی به جای یک کتابخانه ارائه می‌کند.
 
 در سیستم‌های دیگر همیشه یک تماس مسدود کننده برای شروع  event-loop وجود دارد.
```
Comment (Aissaoui-Ahmed) [inline locale/fa/about/index.md]:
```suggestion
در سیستم‌های دیگر همیشه یک تماس مسدود کننده برای شروع event-loop وجود دارد.
```

**Review by Aissaoui-Ahmed — APPROVED** [review]

**Comment by Aissaoui-Ahmed** [comment]
you can add [Unicode](https://en.wikipedia.org/wiki/Arabic_script_in_Unicode) for RegExp

**Comment by nschonni** [comment]
@Aissaoui-Ahmed I was just doing this in VS Code, and I don't think they have much of an extended regex right now. Do you have a suggestion on what to use instead of `\w` for next time?

**Comment by Trott** [comment]
> @Aissaoui-Ahmed I was just doing this in VS Code, and I don't think they have much of an extended regex right now. Do you have a suggestion on what to use instead of `\w` for next time?

Maybe instead of `\w ( )+\w` this?:

```
/\S ( )+\S/
```

**Comment by nschonni** [comment]
@Trott thanks, tried another pass with that

**Comment by XhmikosR** [comment]
I think we should add this regex and the regex to find unneeded escapes as scripts and run them from time to time manually.

**Review by XhmikosR — APPROVED** [review]

**Comment by Trott** [comment]
Markdown and HTML will both treat multiple consecutive spaces in text as a single space. I'm not really sure we need to add tooling for this. Not opposed to a lint rule, but also not really seeing a whole lot of value to going out of our way on this sort of thing. A `prettier`-style option that removes them automatically but doesn't make it the writer's problem to do so might be cool.

**Comment by XhmikosR** [comment]
My suggestion was just so that have a semi-automated way to check this from time to time.

**Pull request merged**